src/main/kotlin/no/nav/familie/ef/mottak/hendelse/JournalføringHendelseConsumer.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.familie.ef.mottak.hendelse
+
+import no.nav.joarkjournalfoeringhendelser.JournalfoeringHendelseRecord
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.springframework.kafka.annotation.KafkaListener
+import org.springframework.kafka.support.Acknowledgment
+import org.springframework.stereotype.Service
+
+
+@Service
+class JournalføringHendelseConsumer(val journalhendelseService: JournalhendelseService) {
+
+    @KafkaListener(id = "familie-ef-mottak",
+                   topics = ["\${JOURNALFOERINGHENDELSE_V1_TOPIC_URL}"],
+                   containerFactory = "kafkaJournalføringHendelseListenerContainerFactory",
+                   idIsGroup = false    )
+    fun listen(consumerRecord: ConsumerRecord<Long, JournalfoeringHendelseRecord>, ack: Acknowledgment) {
+        journalhendelseService.prosesserNyHendelse(consumerRecord, ack)
```

Comment:
Hvordan funker dette hvis vi kjører to instanser av mottak samtidig?

Reply:
prosesserNyHendelse er @Transactional og sjekker om hendelsen er lagret i basen, så det burde fungere fint.

Reply:
Ok, enn så lenge kjører vi bare med én tjeneste også. Får jo se hvordan det oppfører seg med flere.

Reply:
@charliemidtlyng Du tenker på flere instanser som leser fra en og samme logg og behandler de vel? Tror det skal gå fint, men er ikke helt sikker på hvordan det virker eksakt. Eks tilfellet når du har 2 instanser og
instans 1 leser entry 1
instans 2 leser entry 2

instans 1 feiler, instans 2 går fint.. Eller instans 2 commiter til databasen før instans 1.

Reply:
Begge deler, egentlig:
1) Begge instanser plukker "samme" verdi fra køen - den ene skal feile. Hvor ofte vil dette skje - og hvor mye støy vil det generere?
2) Begge instanser plukker hver sin verdi - er rekkefølge på kjøring viktig her? Hva hvis en feiler? 

Dette spør jeg om fordi jeg ikke kan nok om det til å forstå om det "alltid går bra" eller om vi kan brenne oss den dagen vi skrur på flere instanser.

Reply:
Det er jo potensiale for støy, men BA har jo ikke hatt større problemer med det tror jeg.